Dockerfile

```
@@ -8,9 +8,8 @@ WORKDIR /app
 COPY package.json .
 
 RUN apk add --no-cache --virtual .build-dependencies git && \
```

With the updated package.json, npm install now needs python & make (just like in the previous PR, I think it is linked to user-api-client version change ...)
```suggestion
RUN apk add --no-cache --virtual .build-dependencies git python make && \
```